### PR TITLE
Update build settings

### PR DESCRIPTION
**Pull request by Justgo13**
Updates build settings so scenes will switch

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (NosBos) [inline .vscode/settings.json:1]: Could you add `.vscode` to the .gitignore file. It should avoid adding these vscode config things then.

File: TheEthicalHackerCup/ProjectSettings/EditorBuildSettings.asset [hunk 2]
```diff
@@ -4,5 +4,14 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```
Comment (NosBos) [inline TheEthicalHackerCup/ProjectSettings/EditorBuildSettings.asset:7]: Did you add this manually or does this automatically get added somehow?

**Comment by MillanWang** [comment]
Scene switching works with fresh pull. Looks good

File: TheEthicalHackerCup/ProjectSettings/EditorBuildSettings.asset (hunk 2 above)
Comment (Justgo13) [reply TheEthicalHackerCup/ProjectSettings/EditorBuildSettings.asset:7]: In file -> build settings I built after adding the three assets and it created a build for me

**Review by mack-wallace — COMMENTED** [review]
lgtm

**Review by mack-wallace — APPROVED** [review]
approve

**Pull request merged**